apps/charterafrica/src/payload/globals/Footer.js

```diff
@@ -0,0 +1,297 @@
+      label: {
+        en: "Site Description",
+        fr: "Description du site",
+        pt: "Descrição do Site",
+      },
+      type: "richText",
+      localized: true,
+      required: true,
+      admin: {
+        elements: ["h2", "h3", "h4", "h5", "h6", "ol", "ul", "link"],
+      },
+    },
+    {
+      name: "projectDescription",
+      label: {
+        en: "Project Description",
+        pt: "Descrição do Projeto",
+        fr: "Description du projet",
+      },
+      type: "richText",
+      localized: true,
+      required: true,
+      admin: {
+        elements: ["h2", "h3", "h4", "h5", "h6", "ol", "ul", "link"],
+      },
+    },
+    linkArray(),
+    {
+      name: "newsletter",
+      type: "group",
+      label: {
+        en: "Newsletter",
+        fr: "Newsletter",
+        pt: "Boletim de Notícias",
+      },
+      fields: [
+        {
+          name: "title",
+          required: true,
+          localized: true,
+          type: "text",
+          label: {
+            en: "Title",
+            fr: "Titre",
+            pt: "título",
+          },
+        },
+        {
+          name: "embedCode",
+          type: "code",
+          label: {
+            en: "Embed Code",
+            fr: "Code d'intégre",
+            pt: "Código Incorporado",
+          },
+          localized: true,
+          required: true,
+        },
+      ],
+    },
+    {
+      name: "funder",
+      required: true,
+      type: "group",
+      label: {
+        en: "Funder",
+        pt: "Financiador",
+        fr: "Bailleur de fonds",
+      },
+      fields: [
+        {
+          name: "title",
+          required: true,
+          localized: true,
+          type: "text",
+          label: {
+            en: "Footer Title",
+            fr: "Titre",
+            pt: "título",
+          },
+        },
+        {
+          name: "logo",
+          type: "upload",
+          relationTo: "media",
+          required: true,
+          filterOptions: {
+            mimeType: { contains: "image" },
+          },
+          label: {
+            en: "Logo",
+            pt: "Imagem",
+            fr: "Logo",
+          },
+        },
+      ],
+    },
+    {
+      type: "group",
+      name: "contact",
+      label: {
+        en: "Contact",
+        pt: "Contact",
+        fr: "Contact",
+      },
+      admin: {
+        width: "100%",
+      },
+      fields: [
+        {
+          name: "stayInTouch",
+          type: "text",
+          label: {
+            en: "Title",
+            fr: "Titre",
+            pt: "Título",
+          },
+          admin: {
+            description: () =>
+              "Text that appears on contact links e.g Stay in Touch",
+          },
+          localized: true,
+          required: true,
+        },
+        {
+          name: "email",
+          type: "email",
+          label: {
+            en: "Email",
+            pt: "Email",
+            fr: "Email",
+          },
```

Comment:
```suggestion
          name: "email",
          type: "email",
          label: {
            en: "Email",
            pt: "Email",
            fr: "Email",
          },
        ],
      },
      {
        name: "connect",
        type: "group",
        label: {
          en: "Connect",
        },
        fields: [
        {
          name: "title",
          type: "text",
          label: {
            en: "Title",
            fr: "Titre",
            pt: "Título",
          },
          admin: {
            description: () =>
              "Text that appears on contact links e.g Stay in Touch",
          },
          localized: true,
          required: true,
        },
```

Lets separate contact and social media/connect